debug-bar-elasticpress.php

```diff
@@ -89,3 +90,33 @@ function ep_setup_query_log() {
 	EP_Debug_Bar_Query_Log::factory();
 }
 add_action( 'plugins_loaded', 'ep_setup_query_log' );
+
+/**
+ * Display an admin notice if the minimum ElasticPress plugin version is not met.
+ */
+function maybe_display_admin_notice() {
+	if ( ! defined( 'EP_VERSION' ) || version_compare( EP_VERSION, EP_DEBUG_MIN_EP_VERSION, '<' ) ) {
```

Comment:
@burhandodhy As the plugin will start to depend on newer functions coming from the main plugin, if the requirement isn't met we will need to prevent the plugin from being initialized (instead of simply adding a notice), so we will need to tweak the logic a bit or we might end up with a fatal error.